assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -42,10 +43,16 @@ pageStartup.start();
 
 // ----- Redux Store ----- //
 
-const contributionAmount = parseContrib(getQueryParameter('contributionValue'), 'RECURRING').amount;
+const contributionType = parseContrib(getQueryParameter('contribType'), 'MONTHLY');
```

Comment:
Think there might be a subtle bug here. `getQueryParameter` always [returns a lowercase string](https://github.com/guardian/support-frontend/blob/master/assets/helpers/url.js#L5), but `parseContrib` is expecting the strings to be upper case, so it always falls back to the default (`MONTHLY`).

This behaviour for `getQueryParameter` was presumably chosen because there are sometimes case inconsistencies when parameters are passed through (e.g. from dotcom). However, maybe we want to change it?

Reply:
Yes possibly. There did used to be a toUpper() call here to sort this out but it got lost along the way. I've added it back in so the immediate bug is fixed but maybe we should change getQueryParameter?